script/Ciel/Day 9/09_01_CIEL09_6B.txt

```diff
@@ -583,76 +683,89 @@ Weren't you and I supposed to be having some fun? Come on, why don't you give it
+Besides, if you're going to fight a True Ancestor――
+or, more specifically, Arcueid――
+the only option is to take the defensive.
+ Trying to take the lead and go in for the attack would only doom you to certain death.
 }
 [sha:795d164a73edeb5e2e912046f167a4c7cc85b7d8]{
 -- Page 650, Offset 23822.
 -- アルクェイドを本気で撃退するには、“一撃”で彼女を殺せる装備が必要になる。
-To truly drive back Arcueid, a weapon that can kill her 'in a single strike' is required.
+In order to actually stand a chance against Arcueid,
+ you need a weapon that can kill her with a single strike.
 }
 [sha:ae3e4a0516b882053148f4d844b1e65dd8a57002]{
 -- Page 650, Offset 23823.
 -- 少なくともシエルはそう考える。
-At least, that is what Ciel believes.
+That's what Ciel believes, at any rate.
 }
 [sha:a7dfbd446e51cdecce9094f8f42a19d8448df670]{
 -- Page 651, Offset 23824.
 -- 今回はその余裕がなく、また、相手の状態があまりにも異常すぎた。
-This time, she does not currently possess the ability to do that, and her foe's behavior is much too erratic.
+Right now, she doesn't possess such a weapon.
+ On top of that, her foe's behavior is staggeringly unusual.
 }
 [sha:61e657e0330670b78e578352634d9bd5d69b3d8b]{
 -- Page 651, Offset 23825.
 -- もはや彼女に待つのはいたぶり尽くされた後の死しかない。
-All that awaits her now is death, after she's been thoroughly tormented.
+All that awaits her now is torture, followed by death.
 }
 [sha:b8c2235fd9faba86723ebb942d18d61b25225119]{
 -- Page 651, Offset 23826.
 -- その現実を見据えながらも、彼女の胸によぎったものは死への恐怖ではなかった。
-Faced with this truth, it is not the fear of death that fills her heart.
+Even with this fate staring her in the eye,
+ it isn't the fear of death that fills her heart.
 }
 [sha:054356fd3d8223f7e8907308c44267c6cfdaa3ef]{
 -- Page 652, Offset 23827.
 -- “―――こんな、ことって―――”
-'―――I can't―――'
+"I... can't―――"
 }
 [sha:c7688dbe3d67d39275351788b58b50b2ac56b525]{
 -- Page 652, Offset 23828.
 -- たった一撃で内臓のほとんどが壊死している。
-Almost all her inner organs are necrotizing from just a single blow.
+// Necrosis comes up a surprising amount, and feels a bit technical here. Can revert if people feel strongly.
+The single strike she took was enough for most of her organs to start shutting down.
 }
 [sha:6591951385ae9c7b085296b2a9a4bcd496cfed5f]{
 -- Page 652, Offset 23829.
 -- 意識を失うほどの激痛に耐えながら、代行者は歩み寄る吸血姫を凝視する。
-The Executor scrutinizes the approaching vampire while enduring a pain so intense that it threatens to knock her unconscious.
+Even as the excruciating pain threatens to knock her unconscious,
+ the Executor continues to stare at the approaching vampire princess.
 }
 [sha:5b71670527de4f6aaa35a8a2cbcf63c13651275d]{
 -- Page 653, Offset 23830.
 -- “―――信じられない。あの<真|バ><祖|カ>、本当に自分の感情で力を使っていませんか―――!?”
-'―――believe it. Is that stupid True Ancestor really using her power based on her emotions?!'
+// This phrasing to preserve 'emotions', 'feels like it' maybe too weak?
+"―――believe it.
+ Is that stupid True Ancestor really using her power just because her emotions have gotten the better of her?!"
```

Comment:
The current phrasing makes it sound like Arcueid normally just is very selfless, when it's more like; she didn't even have emotions before.

>Is that stupid True Ancestor really using her powers out of her own feelings?!